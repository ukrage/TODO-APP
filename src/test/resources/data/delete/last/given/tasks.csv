id,title,content,finishedFlg
1,タスク,○○を調べる,false
2,タスク2,レポートを作成する,false