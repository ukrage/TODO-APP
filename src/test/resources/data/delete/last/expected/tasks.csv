id,title,content,finishedFlg
1,タスク,○○を調べる,false